Description: libastyle is packaged
  Comment out the build instruction for downloading libastyle
  .
  Placed a "canary" in other download URLs
Author: Geert Stappers <stappers@debian.org>
Last-Updated: 2017-05-27
Forwarded: not needed

Index: arduino/build/build.xml
===================================================================
--- arduino.orig/build/build.xml
+++ arduino/build/build.xml
@@ -455,7 +455,7 @@
 
     <antcall target="unzip">
       <param name="archive_file" value="./libastylej-2.05.1-3.zip" />
-      <param name="archive_url" value="https://downloads.arduino.cc/libastylej-2.05.1-3.zip" />
+      <param name="archive_url" value="https://downloads.arduino.cc/canary_libastylej-2.05.1-3.zip" />
       <param name="final_folder" value="${staging_folder}/libastylej-2.05.1" />
       <param name="dest_folder" value="${staging_folder}" />
     </antcall>
@@ -639,6 +639,10 @@
   </target>
 
   <target name="linux-libastyle-x86" depends="linux-build" description="Download libastyle.so for x86/x64 arch">
+    <!-- (((
+
+	 libastyle is allready packaged
+
     <antcall target="unzip">
       <param name="archive_file" value="./libastylej-2.05.1-3.zip" />
       <param name="archive_url" value="https://downloads.arduino.cc/libastylej-2.05.1-3.zip" />
@@ -647,6 +651,7 @@
     </antcall>
     <copy file="linux/libastylej-2.05.1/libastylej${arch-bits}.so" tofile="linux/work/lib/libastylej.so" />
     <chmod perm="755" file="linux/work/lib/libastylej.so" />
+    ))) -->
 
     <antcall target="portable-${portable}">
       <param name="parentdir" value="linux/work" />
@@ -664,6 +669,10 @@
   </target>
 
   <target name="linux-libastyle-arm" depends="linux-build" description="Download libastyle.so for ARM">
+    <!-- (((
+
+	 libastyle is allready packaged
+
     <antcall target="unzip">
       <param name="archive_file" value="./libastylej-2.05.1-3.zip" />
       <param name="archive_url" value="https://downloads.arduino.cc/libastylej-2.05.1-3.zip" />
@@ -672,6 +681,7 @@
     </antcall>
     <copy file="linux/libastylej-2.05.1/libastylej_arm.so" tofile="linux/work/lib/libastylej.so" />
     <chmod perm="755" file="linux/work/lib/libastylej.so" />
+    ))) -->
 
     <antcall target="portable-${portable}">
       <param name="parentdir" value="linux/work" />
@@ -968,7 +978,7 @@
 
     <antcall target="unzip">
       <param name="archive_file" value="./libastylej-2.05.1-3.zip" />
-      <param name="archive_url" value="https://downloads.arduino.cc/libastylej-2.05.1-3.zip" />
+      <param name="archive_url" value="https://downloads.arduino.cc/canary_libastylej-2.05.1-3.zip" />
       <param name="final_folder" value="${staging_folder}/libastylej-2.05.1" />
       <param name="dest_folder" value="${staging_folder}" />
     </antcall>
